modules/ROOT/pages/_partials/incpg-icr-initialization.adoc

```diff
@@ -124,7 +124,7 @@ include::partial$icr-repl-props-table-sgr2.adoc[]
 
 == Generic Constraints
 
-.Replicating using Inter-Sync{nbsp}Gateway Replication v2
+.Replicating using Inter-Sync{nbsp}Gateway Replication
```

Comment:
```suggestion
.Replication
```
Again, we should not be treating this as a name. It like saying "Replicating using Peer-to-Peer Sync" or "Replication using CBL Sync" ...Please treat it as such throughout the doc